utils/autoargs.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Automatic arguments / retvals display for known library functions.
+ * It's useful especially for libc functions as of now and can be extended to
+ * many other well-known library functions.
+ *
+ * TODO: need to write a script to generate this form from prototypes
+ */
+static char *auto_args_list = "\
+malloc@arg1;free@arg1;\
```

Comment:
C concatenates adjacent string literals into one.  Also you could also make it a const array rather than a pointer variable.  So it could be:

```
static const char auto_arg_list[] =
  "AAA"
  "BBB"
  "CCC"
;
```

Reply:
I changed it as a single string and separated into many pieces of string literals.